testsuite/replay_verify.py

```diff
@@ -93,8 +94,16 @@ def main():
         "TXNS_TO_SKIP",
         "BACKUP_CONFIG_TEMPLATE_PATH",
     ]
+
     if not all(env in os.environ for env in REQUIRED_ENVS):
         raise Exception("Missing required ENV variables")
+    (runner_no, runner_cnt) = (int(sys.argv[1]), int(sys.argv[2])) if len(sys.argv) > 2 else (None, None)
```

Comment:
this is ok for now for 2 args, but any more we might want to use argparse or something